### PR TITLE
Update M-UI

### PR DESCRIPTION
**Pull request by TheFehr**

### CODE REVIEW

**Comment by TheFehr** [comment]
Closes #564

File: frontend/src/layout/DimeAppBar.tsx [hunk 1]
```diff
@@ -94,6 +95,7 @@ class DimeAppBarInner extends React.Component<Props> {
             aria-label={'Menü öffnen'}
             onClick={() => (this.props.mainStore!.drawerOpen = true)}
             className={classNames(classes.menuButton, drawerOpen && classes.menuButtonHidden)}
+            size="large"
```
Comment (lukasbischof) [inline frontend/src/layout/DimeAppBar.tsx:98]: Method now has too many lines

**Comment by floerb** [comment]
Closes #565

**Review by lukasbischof — APPROVED** [review]
delicious 👨‍🍳

**Pull request merged**